2022-08-01 19:02:52,736 INFO    Thread-47 :12723 [wandb_setup.py:_flush():76] Configure stats pid to 12723
2022-08-01 19:02:52,736 INFO    Thread-47 :12723 [wandb_setup.py:_flush():76] Loading settings from /home/kana/.config/wandb/settings
2022-08-01 19:02:52,736 INFO    Thread-47 :12723 [wandb_setup.py:_flush():76] Loading settings from wandb/settings
2022-08-01 19:02:52,736 INFO    Thread-47 :12723 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'noisy_label_cleaning_a', 'entity': 'kka-na', 'root_dir': '/home/kana/Documents/Training/Reliable_Dataset/cleaning/script', 'run_id': '7zlxzese', 'sweep_param_path': '/home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/sweep-7d9wmz9t/config-7zlxzese.yaml', 'sweep_id': '7d9wmz9t'}
2022-08-01 19:02:52,737 WARNING Thread-47 :12723 [wandb_setup.py:_flush():76] Could not save program above cwd: ../src/hyp_tuning.py
2022-08-01 19:02:52,737 INFO    Thread-47 :12723 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': None, 'program': '../src/hyp_tuning.py'}
2022-08-01 19:02:52,737 INFO    Thread-47 :12723 [wandb_init.py:_log_setup():461] Logging user logs to /home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/run-20220801_190252-7zlxzese/logs/debug.log
2022-08-01 19:02:52,737 INFO    Thread-47 :12723 [wandb_init.py:_log_setup():462] Logging internal logs to /home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/run-20220801_190252-7zlxzese/logs/debug-internal.log
2022-08-01 19:02:52,741 INFO    Thread-47 :12723 [wandb_init.py:init():495] calling init triggers
2022-08-01 19:02:52,741 INFO    Thread-47 :12723 [wandb_init.py:init():498] wandb.init called with sweep_config: {'BATCH_SIZE_PER_IMAGE': 256, 'IMS_PER_BATCH': 4, 'iteration': 1000, 'learning_rate': 0.0005026022349544009, 'model': 'COCO-Detection/faster_rcnn_R_101_C4_3x.yaml'}
config: {}
2022-08-01 19:02:52,745 INFO    Thread-47 :12723 [wandb_init.py:init():548] starting backend
2022-08-01 19:02:52,746 INFO    Thread-47 :12723 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-08-01 19:02:52,755 INFO    Thread-47 :12723 [backend.py:ensure_launched():217] starting backend process...
2022-08-01 19:02:52,783 INFO    Thread-47 :12723 [backend.py:ensure_launched():222] started backend process with pid: 26020
2022-08-01 19:02:52,784 INFO    Thread-47 :12723 [wandb_init.py:init():558] backend started and connected
2022-08-01 19:02:52,804 INFO    Thread-47 :12723 [wandb_run.py:_config_callback():1131] config_cb None None {'BATCH_SIZE_PER_IMAGE': 256, 'IMS_PER_BATCH': 4, 'iteration': 1000, 'learning_rate': 0.0005026022349544009, 'model': 'COCO-Detection/faster_rcnn_R_101_C4_3x.yaml'}
2022-08-01 19:02:52,810 INFO    Thread-47 :12723 [wandb_init.py:init():634] updated telemetry
2022-08-01 19:02:52,812 INFO    Thread-47 :12723 [wandb_init.py:init():665] communicating run to backend with 30 second timeout
2022-08-01 19:02:53,620 INFO    Thread-47 :12723 [wandb_run.py:_on_init():1919] communicating current version
2022-08-01 19:02:53,671 INFO    Thread-47 :12723 [wandb_run.py:_on_init():1923] got version response 
2022-08-01 19:02:53,671 INFO    Thread-47 :12723 [wandb_init.py:init():700] starting run threads in backend
2022-08-01 19:02:54,903 INFO    Thread-47 :12723 [wandb_run.py:_console_start():1893] atexit reg
2022-08-01 19:02:54,906 INFO    Thread-47 :12723 [wandb_run.py:_redirect():1766] redirect: SettingsConsole.REDIRECT
2022-08-01 19:02:54,906 INFO    Thread-47 :12723 [wandb_run.py:_redirect():1771] Redirecting console.
2022-08-01 19:02:54,906 ERROR   Thread-47 :12723 [wandb_run.py:_redirect():1830] Failed to redirect.
Traceback (most recent call last):
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1823, in _redirect
    out_redir.install()
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 702, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 652, in add_fd
    self._register()
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 637, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/home/kana/yes/envs/detectron2/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-08-01 19:02:54,906 INFO    Thread-47 :12723 [wandb_init.py:init():732] run started, returning control to user process
2022-08-01 19:02:55,810 INFO    Thread-47 :12723 [wandb_run.py:_finish():1680] finishing run kka-na/noisy_label_cleaning_a/7zlxzese
2022-08-01 19:02:55,811 INFO    Thread-47 :12723 [wandb_run.py:_atexit_cleanup():1862] got exitcode: 1
2022-08-01 19:02:55,811 INFO    Thread-47 :12723 [wandb_run.py:_restore():1834] restore
2022-08-01 19:02:55,815 INFO    Thread-47 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 527
}

2022-08-01 19:02:56,147 INFO    Thread-47 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 527
}

2022-08-01 19:02:56,642 INFO    Thread-47 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5950
}

2022-08-01 19:02:56,744 INFO    Thread-47 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5950
}

2022-08-01 19:02:56,847 INFO    Thread-47 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5950
}

2022-08-01 19:02:56,950 INFO    Thread-47 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5950
}

2022-08-01 19:02:57,052 INFO    Thread-47 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5950
  total_bytes: 5950
}

2022-08-01 19:02:57,155 INFO    Thread-47 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5950
  total_bytes: 5950
}

2022-08-01 19:02:57,257 INFO    Thread-47 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5950
  total_bytes: 5950
}

2022-08-01 19:02:57,361 INFO    Thread-47 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5950
  total_bytes: 5950
}

2022-08-01 19:02:57,464 INFO    Thread-47 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5950
  total_bytes: 5950
}

2022-08-01 19:02:57,566 INFO    Thread-47 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5950
  total_bytes: 5950
}

2022-08-01 19:02:58,255 INFO    Thread-47 :12723 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5950
  total_bytes: 5950
}

2022-08-01 19:02:58,791 INFO    Thread-47 :12723 [wandb_run.py:_on_finish():1991] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5950
  total_bytes: 5950
}
local_info {
}

2022-08-01 19:02:59,915 INFO    Thread-47 :12723 [wandb_run.py:_footer_history_summary_info():3099] rendering history
2022-08-01 19:02:59,916 INFO    Thread-47 :12723 [wandb_run.py:_footer_history_summary_info():3131] rendering summary
2022-08-01 19:02:59,918 INFO    Thread-47 :12723 [wandb_run.py:_footer_sync_info():3055] logging synced files
